### ISSUE TITLE
关于准确率，我训练了5000步后

### ISSUE THREAD

**Issue by liangcaihua** [649453932/Bert-Chinese-Text-Classification-Pytorch#159]
1.下载模型：chinese_L-12_H-768_A-12，并转换为torch

2.使用数据集 THUCNews，并做了如下划分
train_data, test_data, val_data = torch.utils.data.random_split(my_data, [0.99, 0.0001, 0.0099])
划分后的大小为： 训练集，测试集，验证集 (41386, 9, 828)

3. 训练集的第一轮都没跑完，经过约5000步的训练，等不及了

4.最后验证集的准确率为：
avg_acc:0.94674   loss:0.20800

第一次训练，不知道步骤对不对，欢迎交流！

**Comment by liangcaihua** [comment]
参考  http://thuctc.thunlp.org/  上面的说明，准确率貌似差不多了
